src/main/resources/io/jenkins/plugins/appcenter/Messages.properties

```diff
@@ -1,9 +1,11 @@
 AppCenterRecorder.DescriptorImpl.errors.missingApiToken=Please specify an AppCenter API Token
-AppCenterRecorder.DescriptorImpl.errors.invalidApiToken=API Token canot contain whitespace
+AppCenterRecorder.DescriptorImpl.errors.invalidApiToken=API Token can't contain whitespace
 AppCenterRecorder.DescriptorImpl.errors.missingOwnerName=Please specify an AppCenter Owner Name
 AppCenterRecorder.DescriptorImpl.errors.invalidOwnerName=Username can only contain letters, numbers, dashes, underscores, or full stops
 AppCenterRecorder.DescriptorImpl.errors.missingAppName=Please specify an AppCenter App Name
 AppCenterRecorder.DescriptorImpl.errors.invalidAppName=App name cannot contain whitespace
+AppCenterRecorder.DescriptorImpl.errors.missingDistributionGroup=Please specify a Distribution Group
+AppCenterRecorder.DescriptorImpl.errors.invalidDistributionGroup=Distribution Group can only contain letters, numbers, dashes, underscores, or spaces
```

Comment:
I was able to create the distribution group: `:feahifuh[{}[}{{}:\|``¬¬¬¬!"£!"%£^$&%*^(&)` via the AppCenter UI.

So perhaps we do need to support more than what this suggests?

Reply:
Probably. I'd suggest generalising this message and leave the related validator as it is. That leaves us to check if the distribution group value is present and leave the actual validation to AppCenter.